### PR TITLE
Update caliptra-rtl submodule to the latest dev-integrate branch

### LINKED ISSUE #431: ROM should clear SHA512 LOCK explicitly at startup
SHA512 accelerator was recently changed to initialize with LOCK = 1 (see https://github.com/chipsalliance/caliptra-rtl/issues/58).
~ROM currently does not clear the LOCK before trying to use the SHA512 accelerator.~
EDIT: ROM currently attempts to acquire the LOCK before trying to use the SHA512 accelerator, even though it already owns the lock.

### PR DESCRIPTION
**Pull request by korran**

### CODE REVIEW

**Review by jlmahowa-amd — DISMISSED** [review]

**Review by mhatrevi — CHANGES_REQUESTED** [review]

File: kat/src/sha384acc_kat.rs
[the remark this answers is not in the document]
Comment (korran) [reply kat/src/sha384acc_kat.rs:1]: We can't remove this, as there will be a race condition where the sha_acc engine is open for SoC use between drop() being called and report_error().

[the remark this answers is not in the document]
Comment (korran) [reply kat/src/sha384acc_kat.rs:1]: IMHO We probably need to drop this in both cases (so you can unlock it at some well-controlled time later), but that will break the tests as that other unlock code doesn't exist yet.

[the remark this answers is not in the document]
Comment (mhatrevi) [reply kat/src/sha384acc_kat.rs:1]: Fine, we'll double lock (or check if it is already locked and not lock) in report_error.

[the remark this answers is not in the document]
Comment (korran) [reply kat/src/sha384acc_kat.rs:1]: You need to do that anyways. report_error() can be called early in startup while the peripheral is already locked.

**Review by mhatrevi — APPROVED** [review]

**Pull request merged**